llvm/tools/llvm-cas-fuzzer/llvm-cas-fuzzer.cpp

```diff
@@ -0,0 +1,83 @@
+#include "llvm/CAS/ActionCache.h"
+#include "llvm/CAS/BuiltinUnifiedCASDatabases.h"
+#include "llvm/CAS/ObjectStore.h"
+#include "llvm/Support/ThreadPool.h"
+
+using namespace llvm;
+
+extern "C" int LLVMFuzzerInitialize(int *argc, char ***argv) { return 0; }
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  static ExitOnError ExitOnErr("llvm-cas-fuzzer: ");
```

Comment:
I suggest we use `setBanner` to update this string to capture the cas path once we have created the directory since most CAS diagnostics won't contain it themselves (usually it's clear which cas is in use from e.g. the options to the tool).